website/docs/r/schema_template_deploy_ndo.html.markdown

```diff
@@ -41,6 +42,7 @@ resource "mso_schema_template_deploy_ndo" "template_deployer" {
 * This resource is intentionally created non-idempotent so that it deploys the template in every run, it will not fail if there is no change and we deploy or redeploy the template again. When destroying the resource, no action is taken.
 * Prior to deploy or redeploy a schema validation is executed. When schema validation fails, the resource will fail and deploy or redeploy will not be executed.
 * A template can only be undeployed from a site by disassociating the site from the template with the resource mso_schema_site.
+* To adjust the number of retries to ensure successful deployment completion, configure the retries argument in the provider configuration section.
```

Comment:
is there a maximum or recommended amount by us?

Reply:
There is no specific recommendation for the number of retries, as this setting depends on the user's configuration